test/functional/test_runner.py

```diff
@@ -39,7 +39,17 @@
     CROSS = "x "
     CIRCLE = "o "
 
-if os.name != 'nt' or sys.getwindowsversion() >= (10, 0, 14393):
```

Comment:
I wondered if we support Windows earlier than 10.0.14393 (major.minor.build), but it seems we do still support Windows 7 and newer.